wdl-ast/src/validation/imports.rs

```diff
@@ -0,0 +1,89 @@
+//! Validation of imports.
+
+use crate::v1;
+use crate::v1::StringPart;
+use crate::AstNode;
+use crate::Diagnostic;
+use crate::Diagnostics;
+use crate::Document;
+use crate::Span;
+use crate::SupportedVersion;
+use crate::ToSpan;
+use crate::VisitReason;
+use crate::Visitor;
+
+/// Creates an "empty import" diagnostic
+fn empty_import(span: Span) -> Diagnostic {
+    Diagnostic::error("import URI cannot be empty").with_highlight(span)
+}
+
+/// Creates a "placeholder in import" diagnostic
+fn placeholder_in_import(span: Span) -> Diagnostic {
+    Diagnostic::error("import URI cannot contain placeholders")
+        .with_label("remove this placeholder", span)
```

Comment:
should this be:
```suggestion
        .with_fix("remove this placeholder", span)
```
?